azure-proto-core/ResourceContainer.cs

```diff
@@ -19,16 +19,16 @@ namespace azure_proto_core
     ///     The return type of the Creation methods, this can be Response<typeparamref name="T" /> or a
     ///     long-running response
     /// </typeparam>
-    public abstract class ResourceContainerOperations<TOperations, TResource> : OperationsBase
```

Comment:
I think we want to name this ResourceContainerBase to match the pattern of ResourceOperationsBase

Its the concrete classes that need to follow the [ResourceName]Container pattern such as SubscriptionContainer or VirtualMachineContainer